lld/test/ELF/linkerscript/nocrossrefs.test

```diff
@@ -0,0 +1,179 @@
+# REQUIRES: x86
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc --triple=x86_64-unknown-linux -filetype=obj -o main.o main.s
```

Comment:
remove `-unknown-linux`. make it clear this doesn't use Linux specific features.